src/Common/src/Interop/OSX/System.Security.Cryptography.Native.Apple/Interop.PAL_HashAlgorithm.cs

```diff
@@ -0,0 +1,19 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+internal static partial class Interop
+{
+    internal static partial class AppleCrypto
+    {
+        internal enum PAL_HashAlgorithm
+        {
+            Unknown = 0,
+            Md5,
```

Comment:
Yep.  MD5 is already part of .NET Core; and this change is (part of) "switch from OpenSSL to Apple CommonCrypto on macOS"; so no judgment is being applied, just changing providers.  Plus, it probably has ridiculously high usage anyways.